client/main.cpp

```diff
@@ -60,6 +60,7 @@
 #include <sys/wait.h>
 
 #include "client.h"
+#include "platform.h"
```

Comment:
On 12/02/15 14:28, Stephan Kulow wrote:
>> @@ -60,6 +60,7 @@
>>  #include <sys/wait.h>
>>  
>>  #include "client.h"
>> +#include "platform.h"
> 
> I don't have neither platform.h nor determine_platform on linux
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/icecc/icecream/pull/118/files#r24579917
> 

platform.h is services/platform.h

I tested it on Linux, works fine.

??